lib/stylesheet/importer.rb

```diff
@@ -152,8 +152,12 @@ def self.import_color_definitions(theme_id)
         contents << "\n\n"
       end
 
-      if theme_id
-        Theme.list_baked_fields([theme_id], :common, :color_definitions).each do |row|
+      theme_id = theme_id || SiteSetting.default_theme_id
```

Comment:
```suggestion
      theme_id ||= SiteSetting.default_theme_id
```
super super super minor